### DISCUSSION TITLE
Problem with spectra of Radiacode-103

### DISCUSSION THREAD

**Discussion by Rad-zzz-iacode** [Open-Gamma-Project/Gamma-MCA#231] [Q&A]
The site reads spectra fine, but the search for peaks is not accurate, even if you change the settings to 1024 channel.

<img width="518" alt="Снимок экрана 2024-02-08 в 12 37 18" src="https://github.com/OpenGammaProject/Gamma-MCA/assets/98279116/0b4b442b-2b4d-4c89-bd65-ffb6a67ebbc4">
<img width="518" alt="Снимок экрана 2024-02-08 в 12 37 18" src="https://github.com/OpenGammaProject/Gamma-MCA/assets/98279116/41b70661-67ac-420c-9623-d1a445b3cbbe">

**Comment by NuclearPhoenixx** [comment]
Hi there, please attach the file so that I can have a look at the data.

**Comment by Rad-zzz-iacode** [comment]
[Ra-226 .xml.zip](https://github.com/OpenGammaProject/Gamma-MCA/files/14209116/Ra-226.xml.zip)

**Reply by NuclearPhoenixx** [reply]
Thanks for the fast reply! This is not a problem with Gamma MCA, but a problem with your exported file itself. The file contains these calibration coefficients:

```
<Coefficient>-1.387924</Coefficient>
<Coefficient>2.3788693</Coefficient>
<Coefficient>4.0213796E-4</Coefficient>
```

... which all import correctly, so the import is totally fine. Either your calibration is way off, or something went wrong with the file export. There is nothing I can do to fix this, this is something you have to figure out with your Radiacode.

**Comment by Rad-zzz-iacode** [comment]
[Ra-226 _ U .xml.zip](https://github.com/OpenGammaProject/Gamma-MCA/files/14209432/Ra-226._.U.xml.zip)

Here's another file with a different calibration, it doesn't sign some the peaks.

**Reply by NuclearPhoenixx** [reply]
What you can try and do is go to `Settings -> Energy Peak Detection -> Gaussian filter σ` and reduce that from 2 to 1. This makes peak detection a lot more susceptible to noise in the spectrum, but the Radiacode has so few bins that the spectrum is very smooth anyways. I checked again with `Gaussian filter σ` set to `1` and it now detects the other peaks too.

**Reply by Rad-zzz-iacode** [reply]
Can you make a profile in the settings? Radiacode is one of the best selling spectrometers right now, over 15k sold.

**Reply by NuclearPhoenixx** [reply]
Settings are handled on a per-browser basis. There are no profiles and it would be hell of a lot of effort to create the perfect settings for each and every spectrometer out there that people want to use (besides I could never do that on my own just because I don't own everything to test it). If you want to import Radiacode spectra, just remember to decrease the `Gaussian filter σ` setting. I think that's really not too much to ask a user.

If the people at Radiacode increase the bins in some future software update, you'd probably need to change the `Gaussian filter σ` value again. So it's better you as a user understand what you're doing than relying on a profile that could be outdated very quickly. Sorry, but I hope you can understand my point of view.
